pxr/usd/lib/pcp/testenv/testPcpIterator.cpp

```diff
@@ -215,7 +215,7 @@ _TestRandomAccessOperations(IteratorType first, IteratorType last)
 {
     TF_AXIOM(first != last);
 
-    size_t idx = 0;
+    decltype(first - last) idx = 0;
```

Comment:
This one we thought to use std::iterator_traits<IteratorType>::difference_type instead...